testah-junit/src/main/java/org/testah/runner/httpLoad/HttpActor.java

```diff
@@ -73,7 +75,11 @@ public ActorRef getWorkerRouter() {
     }
 
     public static List<ResponseDto> getResults(final Long hashId) {
-        return getResults().get(hashId);
+        HashMap<Long, List<ResponseDto>> resultsLocalPointer = getResults();
+        if (resultsLocalPointer.isEmpty() || !resultsLocalPointer.containsKey(hashId)) {
```

Comment:
you don't need the isEmpty here as containsKey on an empty list will return false